### PR TITLE
Add docs on adding new TSC members

### PR DESCRIPTION
**Pull request by jmertic**
Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

### CODE REVIEW

File: process/tsc_faq.md [hunk 1]
```diff
@@ -66,7 +75,7 @@ For public meetings, LFX PCC provides a public link for sharing the meeting with
 
 ### Agenda
 
-The TSC Chairperson should share an agenda at least 24 hours prior to the meeting, if possible, with an ask to share additional agenda items. Projects can share via the public discussion lists or using a pull request on GitHub.
+The TSC Chairperson should share an agenda at least 24 hours before the meeting with an ask to share additional agenda items. Projects can share via the public discussion lists or a pull request on GitHub.
```
Comment (jfpanisset) [inline process/tsc_faq.md:78]: "Projects can share" -> "Projects can share the agenda".
Also, should we add "via the public discussion lists, posting to the ASWF Slack, or a pull request on GitHub"?

File: process/tsc_faq.md [hunk 2]
```diff
@@ -52,9 +61,9 @@ Roles we've seen in a project include but aren't limited to...
 
 ## Meetings
 
-TSC meetings are opportunities for the project community to come together to share updates and discuss issues and future plans. All TSC meetings are considered open to the public unless there is a sensitive issue to discuss ( in the case of private meetings, Linux Foundation staff should be present at such meetings ).
+TSC meetings are opportunities for the project community to share updates and discuss issues and plans. All TSC meetings are considered open to the public unless there is a sensitive issue to discuss ( in the case of private meetings, Linux Foundation staff should be present at such meetings ).
```
Comment (jfpanisset) [inline process/tsc_faq.md:64]: The part about a LF representative needing to be present at non-public meetings might need a bit of clarification. What would be the process for requesting that? Has it happened before in ASWF?

File: process/tsc_faq.md [hunk 3]
```diff
@@ -17,31 +17,40 @@ The Technical Steering Committee (TSC) is the leadership of the project. This co
...
+[LFX Project Control Center](https://projectadmin.lfx.linuxfoundation.org/) is a tool for managing project committees and roles within a project. Each project will be set up in that tool with a voting committee named 'Technical Steering Committee (TSC),' notating the project chairperson and all voting members. Project chairpersons will have access to manage this in addition to the Linux Foundation staff supporting the project. Additionally, each project should have a [GitHub Team](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams) set up with each of their maintainers, with each project repository granted 'maintainer' permission. Generally, the LF Staff will have this setup for each project, but it's up to the project to maintain after that.
```
Comment (jfpanisset) [inline process/tsc_faq.md:22]: Not sure that's generally known, especially the part about updating the LFX platform. Would be good to review all current projects and make sure that's up to date (unless that's already been happening).

**Review by jfpanisset — DISMISSED** [review]
A few comments that can be addressed later, otherwise all good.

File: process/tsc_faq.md (hunk 1 above)
Comment (jmertic) [reply process/tsc_faq.md]: Cleaned this up to be more agnostic

File: process/tsc_faq.md (hunk 2 above)
Comment (jmertic) [reply process/tsc_faq.md:64]: Not at ASWF, but it has happened in the cases of conduct or legal questions. I would say we would know about the meeting well beforehand ;-)

File: process/tsc_faq.md (hunk 3 above)
Comment (jmertic) [reply process/tsc_faq.md:22]: Yep - we are doing that in the background. Once we transition over then we will make sure TSC leaders are all enabled.

**Comment by jfpanisset** [comment]
Seems good to merge

**Pull request merged**